### ISSUE TITLE
Train my own data, the effect is not good

### ISSUE THREAD

**Issue by jixiaonanzhuaizhuai** [xingyizhou/CenterNet#59]
I copy coco.py to my dataset name and change the category information, and annotation path. During the training, I used a picture and the corresponding json. When the loss converges to about 2, I tested the picture with demo.py, and the effect is very bad. Do I still need to modify other places?

**Comment by xingyizhou** [comment]
Hi
  You can visualize the ground truth and predicted heatmap by `--debug 2`. You will need to add basic dataset information [here](https://github.com/xingyizhou/CenterNet/blob/master/src/lib/utils/debugger.py#L25). Also, make sure your dataset contains sufficient training samples.

**Comment by jixiaonanzhuaizhuai** [comment]
@xingyizhou  Thanks very much! Now the effect is very good for my own data.

**Comment by jixiaonanzhuaizhuai** [comment]
@xingyizhou  When I was training, I found it particularly slow. There was only one picture in my training set, training 90 epochs, and it took 20 minutes. I did not modify any parameters of the model, and tested my image with the model you provided. The effect is good. When I test my image with my own data training model, each object is slightly offset to the right.

**Comment by xingyizhou** [comment]
Good to know it works. But I cannot expect any results if a network is trained with just one sample.

**Comment by zhangtiantian12311** [comment]
@jixiaonanzhuaizhuai when you train on coco data,how to generate image_info_test-dev2017.json,thank you very much
